src/db.test.ts

```diff
@@ -0,0 +1,284 @@
+import { describe, expect, it } from 'vitest';
```

Comment:
It would be good with an integration test which uses a database.

Reply:
I removed the test